data/index.js

```diff
@@ -1,7 +1,7 @@
 var self = require('sdk/self');
 
-// a dummy function, to show how tests work. 
-// to see how to test this function, look at ../test/test-main.js
+// a dummy function, to show how tests work.
+// to see how to test this function, look at test/test-index.js
```

Comment:
Running `jpm init` seems to create the following structure:
![](https://s3.amazonaws.com/f.cl.ly/items/2N2k0I0U3P3K0j0x1f0p/Image%202015-03-05%20at%2010.00.55%20PM.png)
Is this  used outside of init as well?
https://github.com/mozilla/jpm/blob/master/lib/init.js#L56-L57